docs/storing-querying/schemas-in-warehouse/index.md

```diff
@@ -0,0 +1,1663 @@
+
+:::
+
+  </TabItem>
+  <TabItem value="databricks" label="Databricks">
+
+Because the column name for the self-describing event or entity includes the major schema version, each major version of a schema gets a new column:
+
+| Schema | Resulting column |
+|---|---|
+| `com.example/button_press/jsonschema/1-0-0` | `unstruct_event_com_example_button_press_1` |
+| `com.example/button_press/jsonschema/1-2-0` | `unstruct_event_com_example_button_press_1` |
+| `com.example/button_press/jsonschema/2-0-0` | `unstruct_event_com_example_button_press_2` |
+
+When you evolve your schema within the same major version, (non-destructive) changes are applied to the existing column automatically. For example, if you add a new optional field in the schema, a new optional field will be added to the `STRUCT`.
+
+:::info Breaking changes
+
+If you make a breaking schema change (e.g. change a type of a field from a `string` to a `number`) without creating a new major schema version, the loader will not be able to modify the column to accommodate the new data.
+
+In this case, _upon receiving the first event with the offending schema_, the loader will instead create a new column, with a name like `unstruct_event_com_example_button_press_1_0_1_recovered_9999999`, where:
+* `1-0-1` is the version of the offending schema
+* `9999999` is a hash code unique to the schema (i.e. it will change if the schema is overwritten with a different one)
+
+To resolve this situation:
+* Create a new schema version (e.g. `1-0-2`) that reverts the offending changes and is again compatible with the original column. The data for events with that schema will start going to the original column as expected.
+* You might also want to manually adapt the data in the `..._recovered_...` column and copy it to the original one.
+
+Note that this behavior was introduced in RDB Loader 5.3.0.
+
+:::
+
+  </TabItem>
+  <TabItem value="bigquery" label="BigQuery">
+
+Because the column name for the self-describing event or entity includes the full schema version, each version of a schema gets a new column:
+
+| Schema | Resulting column |
+|---|---|
+| `com.example/button_press/jsonschema/1-0-0` | `unstruct_event_com_example_button_press_1_0_0` |
+| `com.example/button_press/jsonschema/1-2-0` | `unstruct_event_com_example_button_press_1_2_0` |
+| `com.example/button_press/jsonschema/2-0-0` | `unstruct_event_com_example_button_press_2_0_0` |
+
+You can use [data models](/docs/modeling-your-data/what-is-data-modeling/index.md) to aggregate the data across multiple columns.
+
```

Comment:
I'd maybe also/alterntively link directly to the macro so that they can use it in their own data models https://github.com/snowplow/dbt-snowplow-utils#combine_column_versions-source